ratpack-cassandra/src/main/java/smartthings/ratpack/cassandra/CassandraModule.java

```diff
@@ -139,8 +161,93 @@ public void setPassword(String password) {
 
 	@Override
 	protected void configure() {
+		bind(Session.class).to(DefaultSession.class).in(Scopes.SINGLETON);
```

Comment:
Your call, what you have is good as well.  I used the optional binder pattern in our internal route common project.  I personally think it makes for a nice extension point, but what you have does as well.

Usage looks like this...
You define a default in a lib module like this:
```
OptionalBinder.newOptionalBinder(binder(), AccountService.class)
    .setDefault().to(DefaultAccountService.class).in(Scopes.SINGLETON);
```

And then you can override it in a project like this:
```
OptionalBinder.newOptionalBinder(binder(), AccountService.class)
    .setBinding().to(ZipkinAccountService.class).in(Scopes.SINGLETON);
```